nilearn/input_data/multi_nifti_labels_masker.py

```diff
@@ -0,0 +1,219 @@
+        Gives which image gives the final shape/size. For example, if
+        `resampling_target` is "data", the atlas is resampled to the
+        shape of the data if needed. If it is "labels" then mask_img
+        and images provided to fit() are resampled to the shape and
+        affine of maps_img. "None" means no resampling: if shapes and
+        affines do not match, a ValueError is raised. Defaults to "data".
+
+    memory: joblib.Memory or str, optional
+        Used to cache the region extraction process.
+        By default, no caching is done. If a string is given, it is the
+        path to the caching directory.
+
+    memory_level: int, optional
+        Aggressiveness of memory caching. The higher the number, the higher
+        the number of functions that will be cached. Zero means no caching.
+
+    n_jobs: integer, optional
+        The number of CPUs to use to do the computation. -1 means
+        'all CPUs', -2 'all CPUs but one', and so on.
+
+    verbose: integer, optional
+        Indicate the level of verbosity. By default, nothing is printed
+
+    strategy: str
+        The name of a valid function to reduce the region with.
+        Must be one of: sum, mean, median, mininum, maximum, variance,
+        standard_deviation
+
+    See also
+    --------
+    nilearn.input_data.NiftiMasker
+    nilearn.input_data.NiftiLabelsMasker
+    """
+
+    def __init__(self, labels_img, background_label=0, mask_img=None,
+                 smoothing_fwhm=None, standardize=False, detrend=False,
+                 low_pass=None, high_pass=None, t_r=None, dtype=None,
+                 resampling_target="data",
+                 memory=Memory(cachedir=None, verbose=0), memory_level=1,
+                 n_jobs=1, verbose=0, strategy="mean"):
+        self.labels_img = labels_img
+        self.background_label = background_label
+        self.mask_img = mask_img
+
+        # Parameters for _smooth_array
+        self.smoothing_fwhm = smoothing_fwhm
+
+        # Parameters for clean()
+        self.standardize = standardize
+        self.detrend = detrend
+        self.low_pass = low_pass
+        self.high_pass = high_pass
+        self.t_r = t_r
+        self.dtype = dtype
+
+        # Parameters for resampling
+        self.resampling_target = resampling_target
+
+        # Parameters for joblib
+        self.memory = memory
+        self.memory_level = memory_level
+        self.n_jobs = n_jobs
+        self.verbose = verbose
+
+        available_reduction_strategies = {'mean', 'median', 'sum',
+                                          'minimum', 'maximum',
+                                          'standard_deviation', 'variance'}
+
+        if strategy not in available_reduction_strategies:
+            raise ValueError(str.format(
+                "Invalid strategy '{}'. Valid strategies are {}.",
+                strategy,
+                available_reduction_strategies
```

Comment:
Instead of `str.format(`, write this as 
```python
"Invalid strategy '{}'. Valid strategies are {}.".format(strategy, available_reduction_strategies)
```
Of course, breaking it down to fit.

Reply:
Done.